src/flat-button.jsx

```diff
@@ -142,20 +142,21 @@ const FlatButton = React.createClass({
       textTransform,
     } = this.constructor.getRelevantContextKeys(this.state.muiTheme);
 
-    const defaultColor = disabled ? disabledTextColor :
+    const defaultTextColor = disabled ? disabledTextColor :
       primary ? primaryTextColor :
       secondary ? secondaryTextColor :
       textColor;
 
-    const defaultHoverColor = ColorManipulator.fade(ColorManipulator.lighten(defaultColor, 0.4), 0.15);
-    const defaultRippleColor = ColorManipulator.fade(defaultColor, 0.8);
+    const filterColor = 'rgb(158,158,158)';
```

Comment:
could you please add this to the theme, use use an existing option? hardcoded values are hard to tackle, some people might not want to follow the specs that closely. @oliviertassinari What do you think?